### DISCUSSION TITLE
add to Repository

### DISCUSSION THREAD

**Discussion by banane-m** [hassio-addons/app-debian-base#131] [Q&A] [answered]
Hello, need your help, I'm new to homeassistant, what's wrong with adding the repository, thanks a lot

https://github.com/hassio-addons/addon-debian-base is not a valid add-on repository

**Comment by frenck** [comment, marked as answer]
This ain't a repository and it ain't an add-on. It provides a base for add-ons. To develop on top of them.

**Comment by banane-m** [comment]
Thanks for the info, I misunderstood

**Comment by banane-m** [comment]
I wanted to install the Debian base as a local addon, but the error came up: 

Add-on could not be installed
COPY failed: file not found in build context or excluded by .dockerignore: stat rootfs: file does not exist

**Reply by frenck** [reply]
> I wanted to install the Debian base as a local addon

As I stated above in my previous answer: **This ain't a repository and it ain't an add-on**
